libraries/AP_NavEKF3/AP_NavEKF3_OptFlowFusion.cpp

```diff
@@ -693,16 +678,16 @@ void NavEKF3_core::FuseOptFlow(const of_elements &ofDataDelayed)
             // correct the covariance P = (I - K*H)*P
             // take advantage of the empty columns in KH to reduce the
             // number of operations
-            for (unsigned i = 0; i<=stateIndexLim; i++) {
-                for (unsigned j = 0; j<=6; j++) {
+            for (uint8_t i = 0; i<=stateIndexLim; i++) {
```

Comment:
The reason these were unsigned is that during early development of the original ekf, it gave slightly faster execution times compared to using a uint_8 for an unknown reason. I would leave these as unsigned in this PR and submit a follow up PR that changes from unsigned to uint8_t in this and all the other fusion methods once we have profiling evidence that the execution time is not adversely affected by doing so.

Reply:
ok thanks. removed this commit to change unsigned to uint8_t